### PR TITLE
Added the babel-inline-react-svg plugin

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Frontend infrastructure

## Description

Adds the [babel-inline-react-svg](https://www.npmjs.com/package/babel-plugin-inline-react-svg) plugin to the application. This is prework for #15311. In #15311, the [react-inline-svg](https://www.npmjs.com/package/react-inlinesvg) package is being used, but it causes issues in tests, and if we mock it (which that PR currently does), we're not really testing it's usage.

It also loads the SVG at runtime as a source in the component, vs. the static import that babel-inline-react-svg provides.

TLDR; the babel-inline-react-svg plugin avoids all those pitfalls.

## Related Tickets & Documents

#15311

## QA Instructions, Screenshots, Recordings

This is putting this plugin in place, but there is nothing to test aside from all tests should pass. Actual testing will occur in #15311.

### UI accessibility concerns?

N/A frontend infrastructure changes only.

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: N/A frontend infrastructure changes only.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

Animated boxes opening revealing rubber ducks

### CODE REVIEW

File: app/javascript/.storybook/main.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ module.exports = {
         ...config.resolve.alias,
         '@crayons': path.resolve(__dirname, '../crayons'),
         '@utilities': path.resolve(__dirname, '../utilities'),
+        '@images': path.resolve(__dirname, '../../assets/images'),
```
Comment (nickytonline) [inline app/javascript/.storybook/main.js:75]: A new webpack alias. It's defined here, environment.js, jsconfig.json as well as in the eslint config.

File: app/javascript/.storybook/main.js [hunk 2]
```diff
@@ -80,4 +81,8 @@ module.exports = {
 
     return config;
   },
+  babel: async (options) => ({
+    ...options,
+    plugins: [...options.plugins, 'inline-react-svg'],
```
Comment (nickytonline) [inline app/javascript/.storybook/main.js:86]: Storybook has its own Babel configuration, so we need to add our changes here as well.

File: app/javascript/.eslintrc.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ module.exports = {
               '@crayons': path.join(__dirname, './crayons'),
               '@utilities': path.join(__dirname, './utilities'),
               '@components': path.join(__dirname, './shared/components'),
+              '@images': path.join(__dirname, '../assets/images'),
```
Comment (nickytonline) [inline app/javascript/.eslintrc.js:31]: I opted for `@images` instead of `@img` @ludwiczakpawel.

**Review by aitchiss — APPROVED** [review]
I double checked that the alias/package is working as expected by importing 

```jsx
import Cog from '@images/cog.svg';
```

and checking this displayed when dropped into the app ✅

**Comment by nickytonline** [comment]
> I double checked that the alias/package is working as expected by importing
> 
> ```js
> import Cog from '@images/cog.svg';
> ```
> 
> and checking this displayed when dropped into the app ✅

I forgot to mention that I have a test branch where I validated all this. 🙃

**Review by maestromac — APPROVED** [review]

**Pull request merged**